### ISSUE TITLE
tint LED-Globeform white+color

### ISSUE THREAD

**Issue by cagapisto** [dresden-elektronik/deconz-rest-plugin#4759]
## Device

- Product name: tint LED-Globeform white+color
- Manufacturer: MLI
- Model identifier: tint-ExtendedColor
- Device type:
  - Light

https://www.mueller-licht.de/produktinformationen/artikel/?q=404036

## Screenshots

###Node Info
![01-Node Info](https://user-images.githubusercontent.com/44967763/114322023-56a65e80-9b1e-11eb-80cb-77d68f057ff6.jpg)

### Basic
![02-Basic Cluster](https://user-images.githubusercontent.com/44967763/114321908-afc1c280-9b1d-11eb-8afa-0c26b1626c36.jpg)

### Identify
![04-Identify Cluster](https://user-images.githubusercontent.com/44967763/114321956-e992c900-9b1d-11eb-93d6-48542eee7f66.jpg)

### Groups
![03-Groups Cluster-2](https://user-images.githubusercontent.com/44967763/114321968-08915b00-9b1e-11eb-8bdc-551963946400.jpg)
![03-Groups Cluster-1](https://user-images.githubusercontent.com/44967763/114321970-09c28800-9b1e-11eb-8a79-7c6a1eee5276.jpg)

### Scenes
![07-Scenes Cluster-3](https://user-images.githubusercontent.com/44967763/114321976-10e99600-9b1e-11eb-80c6-b34b3973eba6.jpg)
![07-Scenes Cluster-2](https://user-images.githubusercontent.com/44967763/114321977-121ac300-9b1e-11eb-8d62-7529d762f406.jpg)

### On/Off
![05-On-Off Cluster-1](https://user-images.githubusercontent.com/44967763/114321982-19da6780-9b1e-11eb-829d-41b74b52c14e.jpg)
![05-On-Off Cluster-2](https://user-images.githubusercontent.com/44967763/114321984-1b0b9480-9b1e-11eb-9e6f-f48aa01ad1d7.jpg)

### Level Control
![06-Level Control Cluster-1](https://user-images.githubusercontent.com/44967763/114321986-1f37b200-9b1e-11eb-9265-096c7cfb2cb5.jpg)
![06-Level Control Cluster-2](https://user-images.githubusercontent.com/44967763/114321987-1fd04880-9b1e-11eb-80b4-0e10534c77aa.jpg)

### Color Control
![08-Color Control Cluster-2](https://user-images.githubusercontent.com/44967763/114321991-25c62980-9b1e-11eb-942d-4b091abc3932.jpg)
![08-Color Control Cluster-3](https://user-images.githubusercontent.com/44967763/114321992-265ec000-9b1e-11eb-8e06-278bd7b328d7.jpg)
![08-Color Control Cluster-4](https://user-images.githubusercontent.com/44967763/114321994-26f75680-9b1e-11eb-9479-7d5d370e788e.jpg)
![08-Color Control Cluster-1](https://user-images.githubusercontent.com/44967763/114321995-278fed00-9b1e-11eb-8305-f40af6687fab.jpg)

### Other clusters that are not mentioned above
![10-Green Power Cluster](https://user-images.githubusercontent.com/44967763/114322007-40989e00-9b1e-11eb-916b-03d04e8c0b26.jpg)
![09-ZLL Commissioning Cluster](https://user-images.githubusercontent.com/44967763/114322008-41313480-9b1e-11eb-8f71-949882f93b41.jpg)

**Comment by cagapisto** [comment]
The device request is still existing, would be great if the issue could be reopened.
If there's anything I can do to support in dealing with the request, I'll gladly help (if it's in my skill set).

**Comment by Mimiix** [comment]
Whats not working on it @cagapisto ?

**Comment by cagapisto** [comment]
I'm sorry, I think I've misread the thread for 'supported devices' in a way as "most lights work with the generic implementation, but 'official' supported devices (those listed in the thread) have a better implementation (regarding color accuracy or such) than the out-of-the-box ones".

If this isn't the case, can I maybe ask why there are some devices on the list of supported devices (e.g. the Müller Licht tint devices), even if they are supported through the generic implementation while others are not listed?
Maybe another stupid question, but smart home devices really aren't one of my strengths: Is there, despite normal usage, a set of tests I can perform with the bulb to see if really everything works correctly?

The only things I found that are not quite working with the bulb are:
- Starting scenes that I've saved
- Schedules for switching the light on or off
I'm mostly using the Hue Essentials App to interact with the bulb, so the problem might be on the side of the App and not with deconz.
